xml/templates/message_templates/payment_or_refund_notification_text.tpl

```diff
@@ -1,5 +1,6 @@
-{if $emailGreeting}{$emailGreeting},
-{/if}{if $isRefund}
+{assign var="greeting" value="{contact.email_greeting}"}{if $greeting}{$greeting},{/if}
```

Comment:
I don't think we should change this one - it already has emailGreeting assigned so we should use it rather than smarty magic

Reply:
I thought this would help consistency. This is the only template {$emailGreeting} is used. Not sure why that is. I think code is easier to read if it is consistent. If you still think it;s better to remove, I will do so.

Reply:
@magnolia61 yes, because  you  would also need to remove where emailGreeting is assigned to the template to remove it from the template